src/tty/unix.rs

```diff
@@ -15,106 +15,76 @@
 //! tty related functionality
 //!
 
-use crate::tty::EventedReadWrite;
+use crate::tty::{EventedReadWrite, EventedPty, ChildEvent, ChildInfo};
 use crate::term::SizeInfo;
 use crate::display::OnResize;
 use crate::config::{Config, Shell};
 use crate::cli::Options;
 use mio;
 
-use libc::{self, c_int, pid_t, winsize, SIGCHLD, TIOCSCTTY, WNOHANG};
+use libc::{self, c_int, winsize, TIOCSCTTY};
+use nix::{ unistd::pipe, pty::openpty, sys::signal::* };
```

Comment:
Why the glob import here? Elements should probably be imported individually and the extra whitespace should be removed.